templates/nro/.circleci/config.yml.tmpl

```diff
@@ -508,27 +490,13 @@ workflows:
       equal: [ "rollback", << pipeline.parameters.release_stage >> ]
     jobs:
       - hold-rollback:
-          <<: *staging_common
```

Comment:
I removed this as `staging_common` (which I renamed to `on_release_tag`) only adds the condition that a job should run on a tag that starts with a "v". But since we already check `equal: [ "rollback", << pipeline.parameters.release_stage >> ]`, it seems we don't need that check.